templates/base.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  {% block title %}
+    <title>Taxi Service</title>
+  {% endblock title %}
+
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  {% load static %}
+  <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+
+</head>
+<body>
+<div class="sidebar">
+  {% include "includes/sidebar.html" %}
+</div>
+{% block content %}
+
+  <div class="content">
```

Comment:
The 'content' div should be inside the 'block content' to ensure that any content blocks defined in child templates are properly wrapped within this div. Consider moving the opening 'div' tag to line 20.